### DISCUSSION TITLE
[Kekulize] Printing own SMILES for Can't kekulize mol.?

### DISCUSSION THREAD

**Discussion by haneul-park** [rdkit/rdkit#7498] [Q&A]
I received the following error messages from `rdkit.standardizer.standardize(, `
and I was wondering which molecules are causing these errors.

```
[18:14:04] Can't kekulize mol.  Unkekulized atoms: 3 5 6 7 9 10
[18:14:04] Can't kekulize mol.  Unkekulized atoms: 3 5 6 7 10 11
[18:14:13] Can't kekulize mol.  Unkekulized atoms: 3 5 25 26 27 28
```

Since I am using a loop of SMILES strings, I cannot know which one is causing the error.
How can I print the original SMILES strings that are causing the kekulization errors?

**Comment by vfscalfani** [comment]
Hi @haneul-park. Can you share some code of what you are trying to do? One basic solution could be to add an index within your loop, something like this with enumerate:

```
mols = []
for idx, smiles in enumerate(smiles_list):
    mol = Chem.MolFromSmiles(smiles)
    if mol is not None:
        # standardization stuff here...
    else:
        print("this mol has an issue:", idx)
```
It's also possible to capture the specific rdkit warning/error if needed:

https://rdkit.org/docs/Cookbook.html#detect-chemistry-problems

http://rdkit.blogspot.com/2016/03/capturing-error-information.html
